tests/automation/user_actions.spec.ts

```diff
@@ -435,3 +435,42 @@ test_Alice_2W(
     );
   },
 );
+
+test_Alice_1W_no_network('Invite a friend', async ({ aliceWindow1, alice }) => {
+  await clickOnTestIdWithText(aliceWindow1, 'new-conversation-button');
+  await clickOnTestIdWithText(aliceWindow1, 'chooser-invite-friend');
+  await waitForTestIdWithText(aliceWindow1, 'your-account-id', alice.accountid);
+  await clickOnTestIdWithText(aliceWindow1, 'copy-button-account-id');
+  // Toast
+  await waitForTestIdWithText(
+    aliceWindow1,
+    'session-toast',
+    englishStrippedStr('copied').toString(),
+  );
+  // Wait for copy to resolve
+  await sleepFor(1000);
+  await waitForMatchingText(
+    aliceWindow1,
+    englishStrippedStr('accountIdCopied').toString(),
+  );
+  await waitForMatchingText(
+    aliceWindow1,
+    englishStrippedStr('shareAccountIdDescriptionCopied').toString(),
+  );
+  // To exit invite a friend
+  await clickOnTestIdWithText(aliceWindow1, 'new-conversation-button');
+  // To create note to self
+  await clickOnTestIdWithText(aliceWindow1, 'new-conversation-button');
+  // New message
+  await clickOnTestIdWithText(aliceWindow1, 'chooser-new-conversation-button');
+  await clickOnTestIdWithText(aliceWindow1, 'new-session-conversation');
+  const isMac = process.platform === 'darwin';
+  await aliceWindow1.keyboard.press(`${isMac ? 'Meta' : 'Control'}+V`);
+  await clickOnTestIdWithText(aliceWindow1, 'next-new-conversation-button');
+  // Did the copied text create note to self?
+  await waitForTestIdWithText(
+    aliceWindow1,
+    'header-conversation-name',
+    'Note to Self',
+  );
+});
```

Comment:
I am not following how that test is a "invite a friend" test.
All we do is open the NTS conversation by pasting our own pubkey. Is that expected for a "invite a friend" test?

Reply:
Its just checking that the invite a friend workflow works. The only thing you can do in that user flow is copy your own account id and send it to someone elsewhere..

Reply:
oh ok. LGTM then